exoscale/loadbalancer.go

```diff
@@ -393,6 +394,9 @@ func getAnnotation(service *v1.Service, annotation, defaultValue string) string
 func getLoadBalancerZone(service *v1.Service) (string, error) {
 	zone, ok := service.Annotations[annotationLoadBalancerZone]
 	if !ok {
+		if defaultZone := os.Getenv("EXOSCALE_DEFAULT_LOADBALANCER_ZONE"); defaultZone != "" {
```

Comment:
At this point, we might as well define the zone for all [zone-local] calls:

```suggestion
		if defaultZone := os.Getenv("EXOSCALE_DEFAULT_ZONE"); defaultZone != "" {
```